### DISCUSSION TITLE
Photopea/Photo Editor Integration?

### DISCUSSION THREAD

**Discussion by WalmartWhiteCat** [ppy/osu#30896] [Ideas] [answered]
With the Skin Editor seeming to be a point of interest to at least some in the dev team, I thought, why not lean into it all the way?

People like having customisation, leading to skins being among the most popular things in the entire game. Alongside that, having the ability to change the look of your entire client is just cool. Furthermore, people like having things only they get to use. The problem with that last part is, 90% of people don't know how to fulfill that, while a remaining 4% likely don't want to pay for Photoshop, its alternatives or a custom skin. And I'd say, even that last percent of people probably don't know a whole lot about Photopea and likely don't even know of its existence.

Why not try and fix that? Why not lean into the customisation aspect that has been so prevalent in osu! for so long now? I know it's not the easiest thing, partnering with a company of any kind; but would it not be incredible to have an on-demand photo editor in the game itself, for people to use whenever they want?

If partnering with a company would be too much, how about collaborating with members of the community, who are so talented and have proven they can help with many things? We have Cyperdark, Redo, iSmileZ, alongside so many others in the skinning community that have an insane amount of talent in GFX. We also have tons and tons of coders with what seems to be limitless passion to push things they care about. With that said, I don't think it would be impossible to get it done.

It won't be a priority for quite a long time from now, I know that. But what about after the initial full release of the game? Why not work to add something like that to osu! and further what the community has already started and what has been prevalent from almost the birth of the game?

I really hope this can be implemented. Even if it's insanely difficult, I don't think it would be a waste of time. People care a lot about the game and the way it looks, as is apparent by the constant pushback in the case of menu design. Why not hand them some extra control and make it easy for everyone to work the client in their own way?

**Comment by peppy** [comment, marked as answer]
Not happening.

This is nightmare-level scope creep for a game.

**Reply by WalmartWhiteCat** [reply]
Fair enough.
